### DISCUSSION TITLE
Latest version available?

### DISCUSSION THREAD

**Discussion by viriatusX** [6c65726f79/Transmissionic#1285] [Q&A] [answered]
Hello!
Is there any way to install the latest version of Transmissionic-GUI after all the latest additions these weeks? The latest ZIP version I see is 1.7.1 from October 2022.

Thank you very much

**Comment by 6c65726f79** [comment, marked as answer]
Hi, the new version will be released very soon, probably this weekend.

**Reply by metaclam** [reply]
hi --- also wondering.  There was an issue apparently fixed ([#1358](https://github.com/6c65726f79/Transmissionic/pull/1358)) that I'm running into.  (on iOS, sideloaded with AltStore). Seems it was committed in June 2023.  Meanwhile it seems the most recent release is 1.8.0 in March 2023.  Is a new release due?  I am not familiar with building it myself.
